Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Good job on implementing stretch and getting all of your font-awesome icons looking good.